charts/karmada/templates/post-delete-job.yaml

```diff
@@ -35,7 +35,7 @@ spec:
             - |
               bash <<'EOF'
               set -ex
-              kubectl delete -f /opt/mount/
+              kubectl delete -f /opt/mount/ --ignore-not-found=true
               kubectl delete cm/{{ $name }}-config -n {{ $namespace }}
```

Comment:
how about line 39? should we add this also?

Reply:
Although the configmap will exist at 99% of the time, but we can add the param alse.